closed/make/modules/openjceplus/Copy.gmk

```diff
@@ -21,6 +21,11 @@
 include $(TOPDIR)/closed/CopySupport.gmk
 
 ifeq (true,$(BUILD_OPENJCEPLUS))
+  # Copy OpenJCEPlus legal files.
+  $(call openj9_copy_files,, \
+	    $(OPENJCEPLUS_TOPDIR)/LICENSE \
+	    $(LEGAL_DST_DIR)/OPENJCEPLUS_LICENSE)
```

Comment:
Please don't mix spaces and tabs here; just use spaces like the rest of the file.